tests/integration/targets/ec2_instance_info/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+  - name: "Make instance in the testing subnet created in the test VPC"
+    ec2_instance:
+      state: present
+      name: "{{ ec2_instance_name }}"
+      image_id: "{{ ec2_ami_id }}"
+      availability_zone: '{{ subnet_b_az }}'
+      tags:
+        TestId: "{{ ec2_instance_tag_TestId }}"
+      user_data: |
+        packages:
+          - httpd
+      instance_type: "{{ ec2_instance_type }}"
+      wait: false
+
+  - name: "Gather {{ ec2_instance_name }} info"
+    ec2_instance_info:
+      filters:
+        "tag:Name": "{{ ec2_instance_name }}"
+      include_attributes:
```

Comment:
This is like you want to test the AWS API, this is not the scope of ansible.
I can removed `enaSupport` for now as it is not supported now

Reply:
Not really. It's ensuring that the choices we include in our ansible module parameters are correct and are passed correctly to boto3. I have already seen examples in other modules where we assume that this is the case and it is in fact not, but we didn't realize it because we don't test every option in the spec.

Reply:
I dont want to do that, I will remove the choices and use the module the same way we are doing for our info modules

Reply:
@hakbailey I have update the tests as required, hope it is ok now